code-studio/src/js/components/share_dialog_body.jsx

```diff
@@ -92,6 +92,16 @@ var ShareDialogBody = React.createClass({
         appType={this.props.appType}/>;
     }
 
+    var exportButton = null;
```

Comment:
Nit: I'm not sure what best pratice is here, but I don't think there's any adverse effect to leaving this undefined, e.g. `var exportButton;`